php_phongo.h

```diff
@@ -47,16 +47,21 @@ ZEND_BEGIN_MODULE_GLOBALS(mongodb)
 	HashTable clients;
 ZEND_END_MODULE_GLOBALS(mongodb)
 
+ZEND_EXTERN_MODULE_GLOBALS(mongodb)
+
 #if PHP_VERSION_ID >= 70000
-#   define MONGODB_G(v) ZEND_MODULE_GLOBALS_ACCESSOR(mongodb, v)
+# define MONGODB_G(v) ZEND_MODULE_GLOBALS_ACCESSOR(mongodb, v)
+# if defined(ZTS) && defined(COMPILE_DL_MONGODB)
+ZEND_TSRMLS_CACHE_EXTERN()
```

Comment:
Good point. I removed `ZEND_EXTERN_MODULE_GLOBALS(mongodb)`. The only time I saw that used in other extensions was in various C files, which may not have included the declaring header.